pkg/distro/rhel9/images.go

```diff
@@ -476,6 +476,11 @@ func edgeRawImage(workload workload.Workload,
 	}
 	img.PartitionTable = pt
 
+	customServiceFile := createFilesystemMounpointService(customizations)
+	if customServiceFile != nil {
+		img.Files = []*fsnode.File{customServiceFile}
```

Comment:
since `img.Files` is an array, and it is already initialized, it does make more sense to me to append to it rather than create a new one

Reply:
You're right.  Even if it's known to be empty at this point (not sure it is), better append to it so a future modification that might end up being added before this line doesn't become a noop.

Reply:
Appended